src/test/scala/org/antipathy/scoozie/builder/TransactionBuilderSpec.scala

```diff
@@ -1,13 +1,13 @@
 package org.antipathy.scoozie.builder
 
-import org.scalatest.{FlatSpec, Matchers}
 import com.typesafe.config.ConfigFactory
-import org.antipathy.scoozie.configuration.Credentials
-import org.antipathy.scoozie.configuration.YarnConfig
-import scala.collection.JavaConverters._
-import scala.collection.immutable._
 import org.antipathy.scoozie.action.control._
+import org.antipathy.scoozie.configuration.{Credentials, YarnConfig}
 import org.antipathy.scoozie.exception._
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.collection.JavaConverters._
+import scala.collection.immutable._
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://app.codacy.com/app/SimonJPegg/scoozie/pullRequest?prid=3072461)